### ISSUE TITLE
Klickbare Stationslinks

### ISSUE THREAD

**Issue by hansmorb** [wielebenwir/commonsbooking#953]
An verschiedenen Orten sollen die Stationsseiten verlinkt sein:

- Item Tabelle
- Karten-Popup
- Location Calendar Header
- Itemliste

**Comment by genevievecory** [comment]
Ich stimme total zu, wollte ich selber machen. Teile habe ich bei uns gebaut, was meinst Du?  z.B.: wp-content/plugins/commonsbooking/src/Model/Item.php


`<?php

/**
 * Template: timeframe-withlocation
 * 
 * This template is included in parent template shortcode-items or shortcode-locations
 *
 * $data is set in parent template
 */


use CommonsBooking\Settings\Settings;

$button_label = \CommonsBooking\Settings\Settings::getOption( COMMONSBOOKING_PLUGIN_SLUG . '_options_templates', 'label-booking-button');
$permalink    = add_query_arg ( 'location', $location->ID, get_the_permalink($item->ID) ); // booking link set to item detail page with location ID
/* customization: $locationlink  */ $locationlink = '/?post_type=cb_location&p=' . $location->ID;
?>

<?php echo commonsbooking_sanitizeHTML($location->thumbnail('cb_listing_small')); // div.thumbnail is printed by function ?>



<div class="cb-list-info">
    <h4 class="cb-title cb-item-title"><a href="<?php echo esc_url($locationlink); ?>"><?php echo commonsbooking_sanitizeHTML($location->post_title); ?></a></h4>
    <div class="cb-dates cb-timeframe-dates">
        <?php
            if(
                array_key_exists('ranges', $data) &&
                count($data['ranges'])
            ) {

                foreach ($data['ranges'] as $range) {
                    echo commonsbooking_sanitizeHTML( \CommonsBooking\Model\Timeframe::formatBookableDate($range['start_date'], $range['end_date']) ) . '<br>';
                }
            }
        ?>
    </div>
</div>
<div class="cb-action">
    <a href="<?php echo esc_url($permalink); ?>" class="cb-button"><?php echo commonsbooking_sanitizeHTML($button_label); ?></a>
</div>
`

**Comment by hansmorb** [comment]
Hallo @genevievecory , ich hab auch gesehen, dass du einige Modifikationen in deinem eigenen Fork vorgenommen hast, erstelle doch gerne einen Pull Request, dann können wir die auch in den master übernehmen :)
